docs/TRANSCRIPTION.md

```diff
@@ -0,0 +1,222 @@
+---
+title: Build a Real-Time Transcription Backend with Astro
+description: Learn how to build a backend to initiate Real Time Transcription using Astro.
+---
+
+50% of Americans watch content with subtitles. Why can't we have this with video calls? Actually, we can do this by using Real-Time Transcription with Agora.
+
+This guide is the third part in a series about building backends with Astro. The first guide covered Token Generation with Astro so we can secure our video calls. We will use the token generation logic in this guide. The second guide was about recording your video to a storage provider of your choice. Real-Time Transcription is a separate feature, so it does not rely on the cloud recording guide, but we will need to generate tokens. Let's get started.
+
+## Prerequisites
+1. NodeJS and Astro installed.
+2. A developer account with [Agora](https://console.agora.io/).
+3. A token generator built with Astro. Find the [guide here](https://github.com/AgoraIO-Community/astro-backend/blob/main/docs/TOKENS.md).
+
+## Project Setup
+This guide will build upon the [token generator guide](https://github.com/AgoraIO-Community/astro-backend/blob/main/docs/TOKENS.md). The token generator guide walks through building the `api/tokens.json` endpoint, which returns a token used to secure your video calls. This endpoint uses a `handleGenerateToken` function that handles token generation logic. Since it is on the same backend server, we will use this function directly within the real-time transcription endpoint.
+
+From the previous guide, you should have `APP_ID` and the `APP_CERTIFICATE` in your environment variables. We will use the `APP_ID` to start and stop the transcription.
+
+## Real Time Transcription Overview
+Real-Time Transcription is a RESTful API provided by Agora that allows you to transcribe your video calls in real time and/or store them within a storage provider of your choice. Your business backend should trigger this transcription service so that sensitive data isn't being passed directly to and from the application.
+
+The flow for a successful Real Time Transcription session is as follows.
+1. Acquire the Agora Real-Time Transcription resource.
+2. Start the Real Time Transcription service.
+3. Stop the Real Time Transcription service.
+
+## Enable Agora RESTful API
```

Comment:
This part is a step the user needs to take in the project setup.

Reply:
This is in the same location as the Cloud Recording Guide

I wrote it this way because project set up, sets up the project before we start building. Then gives overview about what we will build in a general sense, then starts diving into the details of how to build, where this is the first section